docusaurus/docs/welcome/instructions/app-icon.md

```diff
@@ -0,0 +1,177 @@
+3. Right-click the **res** folder and select **New > Image Asset**.
+
+![Image Asset Studio](/img/image-asset-studio.png)
+
+The app icon should have both a square and a round version.
+The tool offers a few options which you can tweak as much as you like, but we only cover the necessary ones.
+
+> ℹ️ The following instructions won't cover how to generate [adaptive icons](https://developer.android.com/guide/practices/ui_guidelines/icon_design_adaptive), which are icons that can display a variety of shapes across different device models.
+> However it isn't much different, actually. To generate those, you should use the **Launcher Icons (Adaptive and Legacy)** option for the **Icon Type**.
+
+> ❗️Android 8.0 Oreo introduced the adaptive icons and apps running on it or later versions should support them.
+
+##### Square icons:
+
+-   **Icon Type** should be the **Launcher Icons (Legacy only)** option.
+-   **Name** should be renamed from `ic_launcher` to `app_icon`.
+-   **Asset Type** should be **Image** and the raw app icon path should be given.
+-   **Shape** should be **Square**.
+
+![Image Asset Studio - Launcher Icons (Legacy only) with Square shape](/img/image-asset-studio-launcher-icons-legacy-only-square.png)
+
+After that you should click on **Next** and finally on the **Finish** button.
+
+##### Round icons:
+
+-   **Icon Type** should be the **Launcher Icons (Legacy only)** option.
+-   **Name** should be renamed from `ic_launcher` to `app_icon_round`.
+-   **Asset Type** should be **Image** and the raw app icon path should be given.
+-   **Shape** should be **Circle**.
+
+![Image Asset Studio - Launcher Icons (Legacy only) with Circle shape](/img/image-asset-studio-launcher-icons-legacy-only-circle.png)
+
+After that you should click on **Next** and finally on the **Finish** button.
+
+> 💡 Note that the Google Play icon won't be generated by the Image Asset Studio, so it should be manually made by following the [official design specifications](https://developer.android.com/google-play/resources/icon-design-specifications).
+
+#### iOS
+
+The iOS assets can be generated using [App Icon Generator](https://appicon.co/).
+On the "iOS and macOS" section, only `iPhone` and `iPad` should be selected.
+It is possible to either click and browse the image source or simply drag and drop it.
+After selecting the image, click on the "Generate" button to download the assets.
+
+Although not mandatory, we recommend to rename the generated assets according to the following pattern: `<resolution>@x<multiplier>`..
+This way, it becomes makes easier to identify the attributes of each icon and their purpose simply by glancing at its name.
+
+> ℹ️ Unfortunately there isn't an official tool for app icon generation on iOS.
+> Even though there are Design related tools that solves this problem, there aren't similar ones which are non-designers friendly and significantly better than the recommended tool.
```

Comment:
A quick search led me to these two developer tools:
- https://github.com/smallmuou/ios-icon-generator
- https://github.com/Fueled/ios-icon-generator

We don't need to cover them extensively, but at least reference them as an alternative.

Reply:
The first one I am sure it isn't suitable. The second one may be worth taking a closer look, will do that. I'm just a bit skeptical due to the last update was done almost 1 year ago

Reply:
The 2nd one works for our case! I will rewrite this part and mention it